frontend/app/components/progress.tsx

```diff
@@ -26,13 +27,19 @@ const indicatorBaseClassName = 'h-full w-full flex-1 transition-all';
 export interface ProgressProps extends React.ComponentPropsWithoutRef<typeof ProgressPrimitive.Root> {
   size?: keyof typeof sizes;
   variant?: keyof typeof variants;
+  label: string;
+  locale: string;
```

Comment:
Don't have to pass `locale` as prop. Use `useTranslation` hook directly into the component.

Example
```tsx
const { i18n } = useTranslation('gcweb');
formatPercent(value, i18n.language);
```